client/src/components/SigninButton.svelte

```diff
@@ -193,7 +193,7 @@
       <List>
         <Item tag="a" href="/settings"><Text>Account Settings</Text></Item>
         {#if userData != undefined}
-          <Item tag="a" href="/stats"><Text>Stats</Text></Item>
+          <Item tag="a" href={`/stats?user=${$user.uid}`}><Text>Stats</Text></Item>
```

Comment:
this could be `href="/stats?user={$user.uid}"`. it doesn't really matter but it would be a little nicer